docs/src/operations/upgrade.md

```diff
@@ -20,11 +20,117 @@ The following procedure describes how to upgrade a broker. If the cluster contai
 
 Note that this procedure results in a downtime of the whole cluster. Currently, a rolling upgrade is not recommended to avoid that an upgraded broker do reprocessing and is affected by the issue.
 
-> TODO: describe how to perform the upgrade on Kubernetes/Helm
+### Upgrade in Kubernetes
+#### Preparing the Upgrade
+##### From broker version < 24.4
 
-### Preparing the Upgrade
+1. Stop the workflow processing
+    * Close all job workers
+    * Interrupt the incoming connections to avoid user commands
```

Comment:
May not be the exact right examples, but it might be good to add some (ingress being one of them), to give some ideas to users how you could interrupt incoming connections.

Reply:
This was just a copy from the other section, should we add this also to the other section then?

Reply:
I guess ingress is pretty k8s specific - for users not using k8s, I can't really tell them how they should do this :sweat_smile: